project/src/test/java/com/google/sps/EventRankerTest.java

```diff
@@ -1,26 +1,48 @@
 package com.google.sps;
 
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
 import com.google.cloud.Date;
 import com.google.sps.data.Event;
 import com.google.sps.data.User;
 import com.google.sps.data.VolunteeringOpportunity;
+import com.google.sps.servlets.EventRankerServlet;
+import com.google.sps.utilities.CommonUtils;
 import com.google.sps.utilities.EventRanker;
+import com.google.sps.utilities.SpannerClient;
+import com.google.sps.utilities.SpannerTasks;
+import com.google.sps.utilities.SpannerTestTasks;
 import com.google.sps.utilities.TestUtils;
 import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import javax.servlet.ServletContextEvent;
```

Comment:
nit: goes after line 26

Reply:
Looking at the linter, I think it's ok with this because lines 25 and 26 are one level deeper and are sorted after this one